libtock/analog_comparator.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "tock.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DRIVER_NUM_ACIFC 0x7
+
+// Does the driver exist?
+int ac_exists(void);
+
+// Request the number of available ACs
+int ac_count(void);
+
+// Compare the voltages of two pins (if one is higher than the other) on the
+// corresponding AC, by polling every second.
+bool ac_comparison(uint8_t);
```

Comment:
What does the argument mean here? The comment says compares two pins but this only takes on argument?